### PR TITLE
[REF][ADD] hr_homeworking : split module in two modules

### PR DESCRIPTION
**Pull request by mepe-odoo**
With this commit, employee's work location can be visible on employee view without calendar. The module hr_homeworking_calendar is a bridge module and is auto-install when calendar is installed.

task: 3642327

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js [hunk 1]
```diff
@@ -9,9 +9,11 @@ const patchHrPresenceStatus = () => ({
     get color() {
         if (this.location) {
             let color = "text-muted";
-            if (this.props.record.data.hr_presence_state !== "to_define") {
+            console.log(this.props.record.data)
+            if (this.props.record.data.hr_presence_state !== "to_define" && this.props.record.data.hr_presence_state !== "offline") {
                 color = this.props.record.data.hr_presence_state === "present" ?  "text-success" : "text-warning";
             }
+            console.log(color)
```
Comment (Bertrand2) [inline addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js:16]: You forgot to remove some logs it seems :)
```suggestion
            if (this.props.record.data.hr_presence_state !== "to_define" && this.props.record.data.hr_presence_state !== "offline") {
                color = this.props.record.data.hr_presence_state === "present" ?  "text-success" : "text-warning";
            }
```

File: addons/hr_homeworking/static/src/components/avatar_card_resource/avatar_card_resource_popover copy.xml
Comment (Bertrand2) [inline addons/hr_homeworking/static/src/components/avatar_card_resource/avatar_card_resource_popover copy.xml:1]: copy? :thinking:

**Review by Bertrand2 — CHANGES_REQUESTED** [review]
There seems to be a lot of functional changes... :thinking: Is that intended? I don't see any information about that on the task.
Also, I notice that some stuff seem "unfinished" as there are still some logs and a copy of a file hanging around :) 
Thanks!

File: addons/hr_homeworking_calendar/static/src/calendar/common/popover/calendar_common_popover.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@
                     </div>
                     <div class="list-group-item">
                         <i class="fa fa-fw fa-user me-2" t-att-class="`wl_color_${props.record.colorIndex}`"/>
-                        <Field name="'employee_name'" record="slot.record"/>
+                        <Field name="'employee_name'" record="slot.record" readonly="1"/>
```
[the remark this answers is not in the document]
Comment (mepe-odoo) [reply addons/hr_homeworking_calendar/static/src/calendar/common/popover/calendar_common_popover.xml:49]: nope it wasn't :D

File: addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const patchHrPresenceStatus = () => ({
     get color() {
         if (this.location) {
             let color = "text-muted";
-            if (this.props.record.data.hr_presence_state !== "to_define") {
+            if (this.props.record.data.hr_presence_state !== "to_define" && this.props.record.data.hr_presence_state !== "offline") {
```
Comment (Bertrand2) [inline addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js:12]: According to the odoo js linter:
```suggestion
            if (
                this.props.record.data.hr_presence_state !== "to_define" &&
                this.props.record.data.hr_presence_state !== "offline"
            ) {
```

**Review by Bertrand2 — CHANGES_REQUESTED** [review]
Changes related to the status icon should be moved in a separate fix commit :+1: 
Thanks! :muscle:

**Review by sofiagvaladze — COMMENTED** [review]
@mepe-odoo this PR is grand combination of split and improvement and fixes, which makes it hard to review.

If it's not too much work, I would like to ask you to separate them in different commits 

So I would like to see separate commit for
1) split
2) imps
3) fixes

File: addons/hr/models/hr_employee_base.py [hunk 4]
```diff
@@ -146,9 +146,9 @@ def _compute_presence_state(self):
         for employee in self:
             state = 'to_define'
             if check_login:
-                if employee.user_id.im_status in ['online', 'leave_online']:
```
[the remark this answers is not in the document]
Comment (mepe-odoo) [reply addons/hr/models/hr_employee_base.py:149]: it's to simplify the heritage :) for example in homeworking you can have office_online or other_online or home_online.

File: addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js [hunk 5]
```diff
@@ -9,7 +9,10 @@ const patchHrPresenceStatus = () => ({
     get color() {
         if (this.location) {
             let color = "text-muted";
-            if (this.props.record.data.hr_presence_state !== "to_define") {
+            if (
+                this.props.record.data.hr_presence_state !== "to_define" &&
+                this.props.record.data.hr_presence_state !== "offline"
```
[the remark this answers is not in the document]
Comment (mepe-odoo) [reply addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js:14]: Yes you're right, I corrected it :)

**Pull request closed without merging**